examples/chip-tool/commands/common/CHIPCommand.cpp

```diff
@@ -29,13 +29,9 @@
 
 using DeviceControllerFactory = chip::Controller::DeviceControllerFactory;
 
-constexpr const char kCommissionerAlpha[] = "alpha";
-constexpr const char kCommissionerBeta[]  = "beta";
-constexpr const char kCommissionerGamma[] = "gamma";
-
-constexpr chip::FabricId kCommissionerAlphaFabricId = 1;
-constexpr chip::FabricId kCommissionerBetaFabricId  = 2;
-constexpr chip::FabricId kCommissionerGammaFabricId = 3;
+constexpr chip::FabricId kIdentityAlphaFabricId = 1;
+constexpr chip::FabricId kIdentityBetaFabricId  = 2;
```

Comment:
nit: rename Beta as Delta and Gamma as Omicron ;)

Reply:
Resolving this comment ;) but solid humor is always appreciated ;)